### PR TITLE
version bump

### PR DESCRIPTION
**Pull request by larryk85**
## Change Description

The bumps the version of CDT.  This will also bump the submodule of antler-proj.

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

**Comment by larryk85** [comment]
Don't merge yet, we still need antler-proj to get a couple of PRs in and then update the submodule.

**Review by dimas1185 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ CDT currently supports Linux x86_64 Debian packages. Visit the [release page](ht
 Download the appropriate version of the Debian package and then install it. To download and install the latest version, run the following:
 
 ```sh
-wget https://github.com/AntelopeIO/cdt/releases/download/v3.1.0/cdt_3.1.0_amd64.deb
-sudo apt install ./cdt_3.1.0_amd64.deb
+wget https://github.com/AntelopeIO/cdt/releases/download/v4.0.0/cdt_4.0.0-rc1_amd64.deb
```
Comment (spoonincode) [inline README.md:18]: I think you need `v4.0.0-rc1` here instead

Comment (larryk85) [reply README.md]: Yup, thanks for catching that.

**Comment by ericpassmore** [comment]
Note:start
group: RELEASE
category: CHORE
summary: Version Bump to CDT 4.0.
Note:end

**Pull request merged**